include/fobnail-attester/meta.h

```diff
@@ -3,37 +3,70 @@
 
 #include <stdint.h>
 
-/* Structure for serial number */
-#define SN_SIZE 64
-struct serial_number {
-    uint8_t size;
-    uint8_t number[SN_SIZE];
+/* Structure for meta data for provisioning and attestation */
+struct meta_data {
+    uint8_t     header_version;
+    uint8_t     mac[6];         /* MAC address */
```

Comment:
Keep it aligned with next line. On the other hand, those fields are self-explanatory, maybe comments can be dropped?
```suggestion
    uint8_t     mac[6];               /* MAC address */
```

Reply:
Fixed